### ISSUE TITLE
System Tables

### ISSUE THREAD

**Issue by tarkal** [ff36/halo-core#5]
<a href="https://github.com/solid-tarka"><img src="https://avatars.githubusercontent.com/u/7042623?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [solid-tarka](https://github.com/solid-tarka)**
_Tuesday Dec 09, 2014 at 16:51 GMT_
_Originally opened as https://github.com/solid-software/phantom-dms/issues/113_

----

System tables from the SMDR spec are enquired in the API EVENTUALLY. However, not until V4/5. Because port forwarding is disabled in phantom configuration will have to be done through the serial port.

**Comment by tarkal** [comment]
This is no longer relevant with the latest design.
